d2b_migrate.module

```diff
@@ -64,6 +71,22 @@ function d2b_migrate_theme($existing, $type, $theme, $path) {
   );
 }
 
+/**
+ * Implements hook_requirements().
+ */
+function d2b_migrate_requirements($phase) {
+  $requirements = array();
+  if ($phase == 'runtime') {
+    $requirements['d2b_migrate'] = array(
+      'title' => 'Drupal 2 Backdrop Migrate',
+      'value' => t('Use with caution'),
+      'severity' => REQUIREMENT_ERROR,
+      'description' => t('This is a development module and should be used with caution.'),
+    );
+  }
+  return $requirements;
```

Comment:
I like this 👍🏼 ...however, it feels like a separate issue and separate PR should created to deal with it.

Reply:
See #14